LICENSE.md

```diff
@@ -19,7 +19,7 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 This software includes Rampastring.Tools and Rampastring.XNAUI. Their license follows:
 
-Copyright (c) 2022 Rami "Rampastring" Pasanen
+Copyright (c) 2016-2023 Rami "Rampastring" Pasanen
```

Comment:
2024

Reply:
No. It's 2023 unless Rampa upgrades his library